src/main.rs

```diff
@@ -169,6 +169,9 @@ struct Args {
         requires = "policy_topic"
     )]
     policy_changes_topic: Option<String>,
+    /// If provided, allows to set a custom Max Execution time for the Host in ms.
+    #[clap(long = "max-execution-time", env = "WASMCLOUD_MAX_EXECUTION_TIME_MS")]
+    max_execution_time: u64,
```

Comment:
We can use existing parsers to make this simpler
```suggestion
    /// If provided, allows to set a custom Max Execution time for the Host in ms.
        #[clap(long = "max-execution-time-ms", default_value = "2000", env = "WASMCLOUD_MAX_EXECUTION_TIME_MS", value_parser = parse_duration)]
    max_execution_time: Duration,
```